### PR TITLE
Add single item selfguided tour

### PR DESCRIPTION
**Pull request by alefaissal**
# Add single item selfguided tour
### This component will be called inside a MAP in the selfguided screen, still missing style for this item

### CODE REVIEW

File: client/js/components/SelfGuidedItem/styles.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {StyleSheet} from 'react-native';
+import {THEME} from '../../config';
+
+const styles = StyleSheet.create({
+  miniTitles: {},
```
Comment (siddharthparmar7) [inline client/js/components/SelfGuidedItem/styles.js:5]: Please choose a more semantic name. ex. subTitle.

File: client/js/components/SelfGuidedItem/styles.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  infobox: {},
+  mainTitle: {},
```
Comment (siddharthparmar7) [inline client/js/components/SelfGuidedItem/styles.js:7]: Please choose a more semantic name. ex. title.

File: client/js/components/SelfGuidedItem/styles.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  location: {},
+});
```
Comment (siddharthparmar7) [inline client/js/components/SelfGuidedItem/styles.js:9]: Please style your component.

File: client/js/components/SelfGuidedItem/SelfGuidedItem.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {TouchableOpacity, Text, View, Image} from 'react-native';
+
+import styles from './styles';
+
+const SelfGuidedItem = ({selfGuidedItem}) => (
+  <TouchableOpacity>
```
Comment (siddharthparmar7) [inline client/js/components/SelfGuidedItem/SelfGuidedItem.js:7]: On touch, where would your user will be navigated to?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Comment by alefaissal** [comment]
### Moving to another branch

**Pull request closed without merging**